File: Month 1/intro to java/day 1/src/Java1/Ques2.java
package Java1;

import java.util.*;

public class Ques2 {
    public static void main(String[] args) {


        Scanner sc = new Scanner(System.in);
        String s = sc.nextLine();
        List<String> list = Arrays.asList(s.split(" "));

        Set<String> uniqueWords = new HashSet<String>(list);
        for (String word : uniqueWords) {
            System.out.println(word + ": " + Collections.frequency(list, word));
        }
    }
}

File: Bootcamp/Android/intro to kotlin 1/src/Ques 2.kt
import java.util.*
import kotlin.collections.HashSet

/*

    Write a program to find the number of occurrences of the duplicate words in a string and print them.


*/

fun main() {

    var scanner = Scanner(System.`in`)
    var inputString = scanner.nextLine()
    var inputStringList = inputString.split(" ").toList()
    val uniqueWords = HashSet(inputStringList)


    for (word in uniqueWords) {
        println("${word}:  ${Collections.frequency(inputStringList, word)}")
    }


}